processing/src/main/java/io/druid/segment/data/RoaringBitmapSerdeFactory.java

```diff
@@ -77,8 +78,12 @@ public BitmapFactory getBitmapFactory()
     }
 
     @Override
+    @Nullable
     public ImmutableBitmap fromByteBuffer(ByteBuffer buffer, int numBytes)
     {
+      if (numBytes == 0) {
```

Comment:
Probably shouldn't be added, `GenericIndexed` should handle nullability with -1's.

Reply:
done.